phpstan-baseline.neon

```diff
@@ -58671,6 +58671,11 @@ parameters:
 			count: 1
 			path: app/bundles/LeadBundle/Tests/EventListener/CompanySubscriberTest.php
 
+		-
+			message: "#^Property Mautic\\\\LeadBundle\\\\Tests\\\\EventListener\\\\Issue9488Test\\:\\:\\$contactRepository \\(Mautic\\\\LeadBundle\\\\Entity\\\\LeadRepository\\) does not accept Doctrine\\\\ORM\\\\EntityRepository\\<Mautic\\\\LeadBundle\\\\Entity\\\\Lead\\>\\.$#"
+			count: 1
+			path: app/bundles/LeadBundle/Tests/EventListener/Issue9488Test.php
+
```

Comment:
Please do not add more tech debt to the baseline. Let's figure out how to make PHPSTAN happy instead. I know it's not that simple with Doctrine's repositories.
```suggestion
```